.github/workflows/publish.yaml

```diff
@@ -8,17 +8,6 @@ on:
   push:
     tags: "[0-9]+.[0-9]+.[0-9]+*"
 
-defaults:
-  run:
-    # Declare bash be used by default in this workflow's "run" steps.
-    #
-    # NOTE: bash will by default run with:
-    #   --noprofile: Ignore ~/.profile etc.
-    #   --norc:      Ignore ~/.bashrc etc.
-    #   -e:          Exit directly on errors
-    #   -o pipefail: Don't mask errors from a command piped into another command
-    shell: bash
-
```

Comment:
I added this in the past, thinking it was relevant - but it wasn't unless we used windows runners.